config/initializers/sidekiq.rb

```diff
@@ -1,9 +1,9 @@
 if defined? Sidekiq
   Sidekiq.configure_server do |config|
-    config.redis = { url: "redis://:#{ENV['REDIS_PASS']}@#{ENV['REDIS_HOST']}:#{ENV['REDIS_PORT']}/1" }
+    config.redis = { url: "redis://#{ENV['REDIS_HOST']}:#{ENV['REDIS_PORT']}/1", id: "0001" }
```

Comment:
I'm not familliar with this setting. I've only ever seen it like this (no id): https://github.com/mperham/sidekiq/wiki/Using-Redis#using-an-initializer  Can you explain why we need this?

Reply:
The jobs aren't getting picked up without it from what I can tell. This ID comes from the terraform output.

Reply:
I'll drop the ID out of this change, as removing the auth is most important...